src/App.tsx

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Filter } from './types/enum/Filter';
+import { TodosState } from './types/TodosState';
+
+const defaultState: TodosState = {
+  todos: [],
+  filter: Filter.All,
+  query: '',
+};
 
 export const App: React.FC = () => {
+  const [todosState, setTodosState] = useState<TodosState>(defaultState);
```

Comment:
better to split it into three states, 'cause they are not connected and can be like separate states